code/tests/functional/backend_api/tests/default/test_conversation_custom.py

```diff
@@ -620,35 +620,3 @@ def test_post_makes_correct_call_to_store_conversation_in_search(
             times=2,
         ),
     )
-
-
-def test_post_returns_error_when_downstream_fails(
```

Comment:
@adamdougal Is it ok if we remove this test as there is no call happening directly to index creation?

Reply:
Ah we still want to test that an error is returned when a downstream fails. We will need to switch this to the first call we're now making instead.